### PR TITLE
syntax tree

### PR DESCRIPTION
**Pull request by xfalcox**
- DEV: Enable ruby Syntax Tree format
- DEV: Apply syntax_tree formatting

### CODE REVIEW

File: app/controllers/discourse_assign/assign_controller.rb [hunk 1]
```diff
@@ -18,16 +19,22 @@ def suggestions
...
+          .assign_allowed
+          .order("X.last_assigned DESC")
+          .limit(6)
 
       render json: {
-        assign_allowed_on_groups: Group.visible_groups(current_user).assign_allowed_groups.pluck(:name),
-        assign_allowed_for_groups: Group.visible_groups(current_user).assignable(current_user).pluck(:name),
-        suggestions: ActiveModel::ArraySerializer.new(users, scope: guardian, each_serializer: BasicUserSerializer),
-      }
+               assign_allowed_on_groups:
```
Comment (CvX) [inline app/controllers/discourse_assign/assign_controller.rb:27]: 😩

File: app/controllers/discourse_assign/assign_controller.rb [hunk 2]
```diff
@@ -72,23 +86,31 @@ def assigned
...
 
-      topics = Topic
-        .includes(:tags)
-        .includes(:user)
-        .joins("JOIN assignments a ON a.target_id = topics.id AND a.target_type = 'Topic' AND a.assigned_to_id IS NOT NULL")
-        .order("a.assigned_to_id, topics.bumped_at desc")
-        .offset(offset)
-        .limit(limit)
+      topics =
+        Topic
```
Comment (CvX) [inline app/controllers/discourse_assign/assign_controller.rb:90]: 😶

File: plugin.rb [hunk 3]
```diff
@@ -53,34 +65,35 @@ class ::Post
...
   DiscoursePluginRegistry.serialized_current_user_fields << frequency_field
-  add_to_serializer(:user, :reminders_frequency) do
-    RemindAssignsFrequencySiteSettings.values
-  end
+  add_to_serializer(:user, :reminders_frequency) { RemindAssignsFrequencySiteSettings.values }
 
   add_to_serializer(:group_show, :assignment_count) do
-    Topic
-      .joins(<<~SQL)
+    Topic.joins(<<~SQL).where(<<~SQL, group_id: object.id).where("topics.deleted_at IS NULL").count
```
Comment (CvX) [inline plugin.rb:93]: uhh

File: plugin.rb [hunk 4]
```diff
@@ -334,11 +389,9 @@ class ::Group
         (assigned_to_id = :user_id AND assigned_to_type = 'User' AND active)
     SQL
 
-    if @options[:filter] != :direct
-      topic_ids_sql << <<~SQL
+    topic_ids_sql << <<~SQL if @options[:filter] != :direct
```
Comment (CvX) [inline plugin.rb:392]: 👀

**Review by pmusaraj — APPROVED** [review]
@CvX per internal discussions I am approving this to unblock @xfalcox, we can discuss stree details internally and report them upstream, the feedback here is useful but not strictly limited to this repo.

**Pull request merged**